pkg/config/otlp.go

```diff
@@ -16,6 +16,7 @@ const (
 	ReceiverSubSectionKey           = "receiver"
 	ExperimentalOTLPReceiverSection = ExperimentalOTLPSection + "." + ReceiverSubSectionKey
 	ExperimentalOTLPMetrics         = ExperimentalOTLPSection + ".metrics"
+	tagCardinalityKey               = "otlp_tag_cardinality"
```

Comment:
Shouldn't this be `ExperimentalOTLPMetrics + ".tag_cardinality"`?

Reply:
I believe this should be changed indeed, the PR description mentions `experimental.otlp.metrics.tag_cardinality`.